### PR TITLE
User, car, hotel, room, booking models are added

### PR DESCRIPTION
**Pull request by Asifkhan-akhan**
all the models are created

### CODE REVIEW

File: models/car.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package models
+
+type Car struct {
```
Comment (OvaisTariq95) [inline models/car.go]: CarRental

File: models/booking.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package models
+
+import "time"
+
+// Booking of a hotel room or a car.
+type Booking struct {
+	ID          int       `json:"id" bson:"id" db:"id" structs:"id"`
+	UserID      int       `json:"user_id" bson:"user_id" db:"user_id" structs:"user_id"`
+	ServiceType string    `json:"service_type" bson:"service_type" db:"service_type" structs:"service_type"` // Either Hotel or Car
```
Comment (OvaisTariq95) [inline models/booking.go:12]: why we need this if we already have service id?

File: models/car.go [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+	ID          int    `json:"id" db:"id" structs:"id" bson:"_id"`
+	Make        string `json:"make" db:"make" structs:"make" bson:"make"`
+	Model       string `json:"model" db:"model" structs:"model" bson:"model"`
```
Comment (OvaisTariq95) [inline models/car.go]: irrelevant

File: models/car.go [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+	Year        int    `json:"year" db:"year" structs:"year" bson:"year"`
```
Comment (OvaisTariq95) [inline models/car.go]: irrelevant

File: models/car.go [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+	Description string `json:"description" db:"description" structs:"description" bson:"description"`
+}
```
Comment (OvaisTariq95) [inline models/car.go:32]: add a stus key aswell for booked/open

**Review by OvaisTariq95 — CHANGES_REQUESTED** [review]
Good Work

File: models/car.go (hunk 1 above)
Comment (Asifkhan-akhan) [reply models/car.go]: changed

File: models/booking.go (hunk 2 above)
Comment (Asifkhan-akhan) [reply models/booking.go:12]: "Booking" is in one-to-one relation with CarRental and Room. so both (CarRental ,Room) can have same ids. So ServiceType will act as a flag between them.

File: models/car.go (hunk 3 above)
Comment (Asifkhan-akhan) [reply models/car.go]: removed

File: models/car.go (hunk 4 above)
Comment (Asifkhan-akhan) [reply models/car.go]: removed

File: models/car.go (hunk 5 above)
Comment (Asifkhan-akhan) [reply models/car.go:32]: added

**Review by OvaisTariq95 — APPROVED** [review]

**Pull request merged**